pom.xml

```diff
@@ -20,57 +20,39 @@
     </distributionManagement>
 
     <properties>
-        <aem.version>6.4.2</aem.version>
-        <sling.version>1.2.2</sling.version>
-        <geronimo.version>1.0</geronimo.version>
+        <io.wcm.aem.version>6.4.2.0000</io.wcm.aem.version>
+        <io.wcm.aem.mock.version>2.5.0</io.wcm.aem.mock.version>
         <acs-commons.version>3.19.0</acs-commons.version>
-        <servlet.api.version>3.1.0</servlet.api.version>
-        <jsp.api.version>2.1</jsp.api.version>
-        <jcr.version>2.0</jcr.version>
         <taglib.version>5.7.4</taglib.version>
         <guava.version>22.0</guava.version>
-        <apache.commons.lang.version>3.5</apache.commons.lang.version>
         <jackson.version>2.8.4</jackson.version>
```

Comment:
It looks like `io.wcm.maven.aem-dependencies` includes jackson, and we could use the version from there.

Reply:
Yes in github appears the jackson dependencies but check this (see [here](http://central.maven.org/maven2/io/wcm/maven/io.wcm.maven.aem-dependencies/6.4.2.0000/io.wcm.maven.aem-dependencies-6.4.2.0000.pom) the release version does not containts the jackson dependency